cloud/aws/services/certificates/certificates.go

```diff
@@ -193,10 +193,16 @@ func EncodePrivateKeyPEM(key *rsa.PrivateKey) []byte {
 
 func DecodeCertPEM(encoded []byte) (*x509.Certificate, error) {
 	block, _ := pem.Decode(encoded)
+	if block.Bytes == nil {
+		return nil, nil
+	}
 	return x509.ParseCertificate(block.Bytes)
 }
 
 func DecodePrivateKeyPEM(encoded []byte) (*rsa.PrivateKey, error) {
 	block, _ := pem.Decode(encoded)
```

Comment:
Should we log out the error here and above?

Reply:
Potentially, that was going to be my first approach, but then I figured since we are returning a pointer to a certificate/key that it might make sense to just return nil instead to indicate no certificate/key could be decoded from input...

I can go either way with it, though.

Reply:
That sounds good, we should put that in a comment